src/sdam/server.ts

```diff
@@ -234,7 +257,9 @@ export class Server extends TypedEventEmitter<ServerEvents> {
    * this will be a no-op.
    */
   requestCheck(): void {
-    this[kMonitor].requestCheck();
+    if (!this.s.options.loadBalanced) {
```

Comment:
Very minor thing, but I guess for consistency with the diff immediately above, maybe this could be 
```suggestion
    if (!this.loadBalanced) {
```

throughout the file?

Reply:
Thanks for catching that. I've updated.